current_spring2026/submit_graphrag.sh

```
@@ -0,0 +1,46 @@
+#!/bin/bash -l
+#$ -P sparkgrp
+#$ -l h_rt=4:00:00
+#$ -N bpl-graph-build
+#$ -j y
+#$ -o logs/graph-build.log
+#$ -l gpus=1
+#$ -l gpu_memory=16G
+#$ -l gpu_c=7.5
+#$ -pe omp 8
+
+source /etc/profile.d/modules.sh
```

Shellcheck (0.11.0)

[info] Not following: /etc/profile.d/modules.sh was not specified as input (see shellcheck -x).

(SC1091)